sdk/eventhub/Microsoft.Azure.EventHubs/assets/live-tests-azure-setup.ps1

```diff
@@ -0,0 +1,383 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
Is there a unified way to do this across different client libraries running live services? I remember reading a thread in Teams, but curious why Powershell over ARM templates?

Reply:
Service Principals and permissions are not, as far as I know or was able to determine, available in ARM.... so, Powershell.  

There's been a little bit of chatter about centralizing, but I don't think the approach or concept is fully baked yet and was mostly centered around the more standard needs of services using recorded tests with the HTTP pipeline, at present.